src/NuGet.Core/NuGet.Packaging/ContentModel/ManagedCodeConventions.cs

```diff
@@ -502,6 +507,17 @@ internal ManagedCodePatterns(ManagedCodeConventions conventions)
                     {
                         new PatternDefinition("contentFiles/{codeLanguage}/{tfm}/{any?}"),
                     });
+
+                ToolsAssemblies = new PatternSet(
+                    conventions.Properties,
+                    groupPatterns: new PatternDefinition[]
+                        {
+                            new PatternDefinition("tools/{tfm}/{rid}/{any?}", table: DotnetAnyTable),
+                        },
+                    pathPatterns: new PatternDefinition[]
+                        {
+                            new PatternDefinition("tools/{tfm}/{rid}/{any}", table: DotnetAnyTable),
```

Comment:
Use `DefaultTfmAny` instead of `DotnetAnyTable` for both. At this point `dotnet` is deprecated so it would cause problems to use that for netstandard. 

You also want the real `Any` like `build/` uses.
 
You'll need to update the tests around this also.